site/content/contribute/getting-started/code-review.md

```diff
@@ -0,0 +1,108 @@
+5. Wait for your code to be merged.
+    * Larger pull requests may require more time to review.
+    * Once all reviewers have approved your changes, they will handle merging your code.
+
+If you are a core committer seeking a review
+--------------------------------------------
+
+1. Submit your pull request.
+    * Follow the [contribution checklist](https://developers.mattermost.com/contribute/getting-started/contribution-checklist/).
+2. Assign a [product manager](/contribute/getting-started/core-committers/#product-managers) to your review and label your pull request with `1: PM Review`.
+    * If your changes do not affect the user experience, you may skip this step.
+    * Product managers ensure the changes meet [user experience guidelines](https://docs.mattermost.com/developer/fx-guidelines.html).
+    * Wait for their review before continuing so as to avoid churn if changes are requested.
+    * Note that product managers may assign core committers after completing their own review.
+3. Assign two [core committers](/contribute/getting-started/core-committers/) to your review and label your pull request with `2: Dev Review`.
+    * When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
+    * When picking your second core committer, consider someone unrelated to your changes. A fresh and unbiased set of eyes can be invaluable, and exposing the team to new parts of the code helps spread out domain knowledge.
+    * Don't be afraid to pick someone who gives "hard" reviews. Code review feedback is never a personal attack: it should "sharpen" the skills of both the author and the reviewers, not to mention improving the quality of the product.
+    * If you are Mattermost staff, try to take into account the timeoff calendar.
+    * Try to avoid assigning the same person to all of your reviews unless they are related.
+    * When in doubt, ask for recommendations on our community server.
+4. Optionally assign a [QA tester](/contribute/getting-started/core-committers/#qa-testers) and label your pull request with `QA Review`.
+    * Not every pull request requires QA Review. Prioritize large features or material changes to the product for QA review.
+    * Pull requests labelled with `QA Review` automatically notify the QA team at large.
+    * Reviews by QA may occur at the same time as review by core committers.
+5. Apply additional labels as necessary:
+    * `CherryPick/Approved`: Apply this if the pull request is meant for a quality or patch release.
+    * `Do Not Merge/Awaiting PR`: Apply this if the pull request depends on another (e.g. server changes)
+    * `Setup Test Server`: Apply this to create a test server with your changes for review.
+    * See [labels](/contribute/getting-started/labels) for more details.
+6. Assign a milestone as necessary.
```

Comment:
Why would you not assign a milestone? At least for reviewer it's more clear what the expectations are and what the "deadline" is.

Reply:
My impression is that not everything is "scheduled" for a given release -- sometimes that decision is deferred while a review is ongoing. Omitting a milestone essentially amounts to a signal indicating a somewhat lower priority, though I don't think that changes any of the SLAs around required feedback.

Reply:
Agree. It's better to not create additional pressure for milestone and keep things flexible.